### ISSUE TITLE
[1.14.4] Server having issues loading machine recipes

### ISSUE THREAD

**Issue by ToukaDouka** [TechReborn/TechReborn#1876]
Seems something in reborncore is broken, preventing the server from loading machine recipes.

Minecraft Version 1.14.4
fabric-api-0.4.1+build.245-1.14
RebornCore-1.14.4-4.0.20+build.190
TechReborn-1.14.4-3.0.18+build.256
fabric-server-launch Loader Version 0.5.3+build.168

**Initial error**
```
[15:33:42] [Server thread/INFO]: [STDERR]: java.lang.BootstrapMethodError: java.lang.NoSuchMethodError: net.minecraft.class_2370.method_10250(Lnet/minecraft/class_2960;)Z
[15:33:42] [Server thread/INFO]: [STDERR]:      at reborncore.common.crafting.ConditionManager.<clinit>(ConditionManager.java:24)
[15:33:42] [Server thread/INFO]: [STDERR]:      at reborncore.common.crafting.RebornRecipeType.read(RebornRecipeType.java:66)
[15:33:42] [Server thread/INFO]: [STDERR]:      at reborncore.common.crafting.RebornRecipeType.method_8121(RebornRecipeType.java:45)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1863.method_17720(class_1863.java:115)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1863.method_20705(class_1863.java:53)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1863.method_18788(class_1863.java:34)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_4080.method_18790(class_4080.java:13)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.util.concurrent.CompletableFuture.uniAccept(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_4014.method_18365(class_4014.java:71)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_3738.run(class_3738.java:18)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1255.method_18859(class_1255.java:144)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_4093.method_18859(class_4093.java:23)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1255.method_16075(class_1255.java:118)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:712)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:706)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1255.method_18857(class_1255.java:127)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_3752(MinecraftServer.java:1471)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_3800(MinecraftServer.java:458)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_3735(MinecraftServer.java:374)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_3176.method_3823(class_3176.java:218)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:615)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.lang.Thread.run(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]: Caused by: java.lang.NoSuchMethodError: net.minecraft.class_2370.method_10250(Lnet/minecraft/class_2960;)Z
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MethodHandleNatives.resolve(Native Method)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MemberName$Factory.resolve(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MemberName$Factory.resolveOrFail(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MethodHandles$Lookup.resolveOrFail(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MethodHandles$Lookup.linkMethodHandleConstant(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MethodHandleNatives.linkMethodHandleConstant(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      ... 24 more
```

**Following errors loading machine recipes all pretty much look the same as the one below**
```
[15:33:42] [Server thread/ERROR]: Failed to read recipe: techreborn:blast_furnace/steel_ingot_from_refined_iron
[15:33:42] [Server thread/INFO]: [STDERR]: java.lang.NoClassDefFoundError: Could not initialize class reborncore.common.crafting.ConditionManager
[15:33:42] [Server thread/INFO]: [STDERR]:      at reborncore.common.crafting.RebornRecipeType.read(RebornRecipeType.java:66)
[15:33:42] [Server thread/INFO]: [STDERR]:      at reborncore.common.crafting.RebornRecipeType.method_8121(RebornRecipeType.java:45)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1863.method_17720(class_1863.java:115)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1863.method_20705(class_1863.java:53)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1863.method_18788(class_1863.java:34)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_4080.method_18790(class_4080.java:13)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.util.concurrent.CompletableFuture.uniAccept(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_4014.method_18365(class_4014.java:71)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_3738.run(class_3738.java:18)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1255.method_18859(class_1255.java:144)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_4093.method_18859(class_4093.java:23)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1255.method_16075(class_1255.java:118)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:712)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:706)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1255.method_18857(class_1255.java:127)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_3752(MinecraftServer.java:1471)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_3800(MinecraftServer.java:458)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_3735(MinecraftServer.java:374)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_3176.method_3823(class_3176.java:218)
[15:33:42] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:615)
[15:33:42] [Server thread/INFO]: [STDERR]:      at java.lang.Thread.run(Unknown Source)
```

**Comment by Kjasi** [comment]
I, too, am having this issue when starting my server.

Here's my server's log:
```
[20:21:35] [main/INFO]: Loading for game Minecraft 1.14.4
[20:21:43] [main/WARN]: Mod 'reborncore' (4.0.20+build.190) uses 'requires' key in fabric.mod.json, which is not supported - use 'depends'
[20:21:43] [main/INFO]: [FabricLoader] Loading 44 mods: minecraft@1.14.4, fabric-renderer-api-v1@0.1.1+591e97ae42, fabric-networking-blockentity-v0@0.2.0+c877038942, fabric-keybindings-v0@0.1.1+591e97ae42, terrestria@1.0.11+build.40, inventorysorter@1.5.7, org_spongepowered_configurate-hocon@3.6, fabricloader@0.6.3+build.167, fabric-renderer-indigo@0.1.13+591e97ae42, fabric-containers-v0@0.1.2+591e97ae42, fabric-dimensions-v1@0.1.0+369ab22e42, fabric-biomes-v1@0.1.0+591e97ae42, fabric-crash-report-info-v1@0.1.1+591e97ae42, fabric-events-interaction-v0@0.1.1+591e97ae42, fabric-api-base@0.1.0+2983bc0442, reborncore@4.0.20+build.190, fabric-rendering-v0@0.1.1+591e97ae42, cloth-config2@1.1.2, fabric-rendering-data-attachment-v1@0.1.1+c877038942, com_typesafe_config@1.3.1, fabric-resource-loader-v0@0.1.3+591e97ae42, fabric-textures-v0@0.1.4+591e97ae42, expandedstorage@3.6.29+1.14.4, fabric-content-registries-v0@0.1.1+591e97ae42, fabric-tag-extensions-v0@0.1.1+591e97ae42, creativebuttonmover@0.1.1+1.14.4, blue_endless_jankson@+, org_spongepowered_configurate-core@3.6, fabric-rendering-fluids-v1@0.1.2+36f27aa342, fabric-registry-sync-v0@0.2.2+591e97ae42, fabric-commands-v0@0.1.1+591e97ae42, traverse@2.0.8+build.26, fabric-mining-levels-v0@0.1.0+591e97ae42, fabric-events-lifecycle-v0@0.1.1+591e97ae42, fabric-loot-tables-v1@0.1.0+591e97ae42, fabric@0.4.0+build.240-1.14, fabric-particles-v1@0.1.1+c877038942, fabric-item-groups-v0@0.1.0+591e97ae42, fabric-models-v0@0.1.0+591e97ae42, team_reborn_energy@0.0.7, terraform@1.1.8+build.15, fabric-object-builders-v0@0.1.1+591e97ae42, fabric-networking-v0@0.1.3+591e97ae42, techreborn@3.0.18+build.256
[20:21:43] [main/WARN]: Mod `blue_endless_jankson` (+) does not respect SemVer - comparison support is limited.
[20:21:43] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.7.11 Source=file:/D:/Minecraft%20Servers/My%20Server/fabric-server-launch.jar Service=Knot/Fabric Env=SERVER
[20:21:43] [main/INFO]: Compatibility level set to JAVA_8
[20:21:46] [main/WARN]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[20:21:46] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[20:21:46] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[20:21:46] [main/WARN]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[20:21:46] [main/WARN]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[20:21:47] [main/INFO]: Reborn core is done for now, now to let other mods have their turn...
[20:21:47] [main/INFO]: Loaded torus size map in 77ms
[20:21:47] [main/INFO]: TechReborn setup done!
[20:21:47] [Server thread/INFO]: Starting minecraft server version 1.14.4
[20:21:47] [Server thread/INFO]: Loading properties
[20:21:47] [Server thread/INFO]: Default game type: SURVIVAL
[20:21:47] [Server thread/INFO]: Generating keypair
[20:21:47] [Server thread/INFO]: Starting Minecraft server on *:25566
[20:21:47] [Server thread/INFO]: Using default channel type
[20:21:48] [Server thread/INFO]: Preparing level "My Server"
[20:21:48] [Server thread/INFO]: [fabric-registry-sync] Loaded registry data [file 1/4]
[20:21:48] [Server thread/INFO]: Reloading ResourceManager: Default, ExpandedStorage, Fabric Mining Levels (v0), Terrestria, Traverse, Tech Reborn
[20:21:49] [Server thread/INFO]: [STDERR]: java.lang.BootstrapMethodError: java.lang.NoSuchMethodError: net.minecraft.class_2370.method_10250(Lnet/minecraft/class_2960;)Z
[20:21:49] [Server thread/INFO]: [STDERR]:      at reborncore.common.crafting.ConditionManager.<clinit>(ConditionManager.java:24)
[20:21:49] [Server thread/INFO]: [STDERR]:      at reborncore.common.crafting.RebornRecipeType.read(RebornRecipeType.java:66)
[20:21:49] [Server thread/INFO]: [STDERR]:      at reborncore.common.crafting.RebornRecipeType.method_8121(RebornRecipeType.java:45)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1863.method_17720(class_1863.java:115)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1863.method_20705(class_1863.java:53)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1863.method_18788(class_1863.java:34)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_4080.method_18790(class_4080.java:13)
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.util.concurrent.CompletableFuture.uniAccept(Unknown Source)
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.util.concurrent.CompletableFuture$UniAccept.tryFire(Unknown Source)
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_4014.method_18365(class_4014.java:71)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_3738.run(class_3738.java:18)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1255.method_18859(class_1255.java:144)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_4093.method_18859(class_4093.java:23)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1255.method_16075(class_1255.java:118)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:712)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:706)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_1255.method_18857(class_1255.java:127)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_3752(MinecraftServer.java:1471)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_3800(MinecraftServer.java:458)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.method_3735(MinecraftServer.java:374)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.class_3176.method_3823(class_3176.java:218)
[20:21:49] [Server thread/INFO]: [STDERR]:      at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:615)
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.lang.Thread.run(Unknown Source)
[20:21:49] [Server thread/INFO]: [STDERR]: Caused by: java.lang.NoSuchMethodError: net.minecraft.class_2370.method_10250(Lnet/minecraft/class_2960;)Z
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MethodHandleNatives.resolve(Native Method)
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MemberName$Factory.resolve(Unknown Source)
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MemberName$Factory.resolveOrFail(Unknown Source)
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MethodHandles$Lookup.resolveOrFail(Unknown Source)
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MethodHandles$Lookup.linkMethodHandleConstant(Unknown Source)
[20:21:49] [Server thread/INFO]: [STDERR]:      at java.lang.invoke.MethodHandleNatives.linkMethodHandleConstant(Unknown Source)
[20:21:49] [Server thread/INFO]: [STDERR]:      ... 24 more```

**Comment by modmuss50** [comment]
Should be fixed in RebornCore 4.0.21, let me know if you still have issues with that.
